runtime/builtin_types_test.go

```diff
@@ -23,6 +23,33 @@ import (
 	"testing"
 )
 
+func TestBuiltinDelAttr(t *testing.T) {
+	fun := wrapFuncForTest(func(f *Frame, args ...*Object) (*Object, *BaseException) {
+		fooObject := newTestClass("Foo", []*Type{ObjectType}, toDictUnsafe(args[0])).ToObject()
```

Comment:
If I create objects inside TestBuiltinDelAttr, I thini delAttr or setAttr on one test may modify the object and it will effect another tests. Tests have to run in particular order for that case, I guess.
Is that acceptable?